### PR TITLE
pkgctl: update page; pkgctl-{build,db-update,release}: add page; pkgctl-version: remove page

### LINKED ISSUE #10590: Let's Document: pkgctl
https://man.archlinux.org/man/pkgctl.1

## Commands

- [x] pkgctl → #10573
- [x] pkgctl auth → #10595
- [ ] pkgctl build → #10808
- [x] pkgctl diff → #10599
- [ ] pkgctl release → #10808
- [x] pkgctl repo → #10598
- [x] pkgctl version → #10600

### PR DESCRIPTION
**Pull request by kbdharun**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

Closes #10590

### CODE REVIEW

File: pages/linux/pkgctl-db-update.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# pkgctl db update
+
+> Update the pacman database as final release step for packages that have been transfered and staged on <repos.archlinux.org>.
```
Comment (acuteenvy) [inline pages/linux/pkgctl-db-update.md:3]:
```suggestion
> Update the pacman database as final release step for packages that have been transfered and staged on <https://repos.archlinux.org>.
```

File: pages/linux/pkgctl-db-update.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+> More information: <https://man.archlinux.org/man/pkgctl-build.1>.
```
Comment (acuteenvy) [inline pages/linux/pkgctl-db-update.md:4]: Wrong man page.
```suggestion
> More information: <https://man.archlinux.org/man/pkgctl-db-update.1>.
```

File: pages/linux/pkgctl.md [hunk 3]
```diff
@@ -3,6 +3,30 @@
...
+
+`tldr pkgctl-diff`
+
+- View documentation for releasing build artifacts:
+
+`tldr pkgctl-release`
+
+- View documentation for managing Git packaging repositories and their configuration:
+
+`tldr pkgctl-repo`
```
Comment (acuteenvy) [inline pages/linux/pkgctl.md:28]:
```suggestion
`tldr pkgctl auth`

- View documentation for building packages inside a clean `chroot`:

`tldr pkgctl build`

- View documentation for updating the binary repository as final release step:

`tldr pkgctl db update`

- View documentation for comparing package files using different modes:

`tldr pkgctl diff`

- View documentation for releasing build artifacts:

`tldr pkgctl release`

- View documentation for managing Git packaging repositories and their configuration:

`tldr pkgctl repo`
```

File: pages/linux/pkgctl.md [hunk 4]
```diff
@@ -3,6 +3,30 @@
 > Unified command-line devtools frontend for Arch Linux.
```
Comment (acuteenvy) [inline pages/linux/pkgctl.md:3]:
```suggestion
> Unified command-line frontend for Arch Linux devtools.
```

File: pages/linux/pkgctl-db-update.md (hunk 2 above)
Comment (kbdharun) [reply pages/linux/pkgctl-db-update.md:4]: Nice catch, will apply the suggestions. Thanks.

**Review by acuteenvy — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**